.changeset/old-sheep-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-frontend/application-components': patch
+---
+
+Fix for the `PageContentWide` warning
```

Comment:
Maybe we can provide some more details here:
```suggestion
In version `22.3.3` we introduced a bug in the `PageContentWide` component leading to it sending warning messages to browser console log when correctly using it.

We're now fixing that incorrect behaviour.
```
```

Reply:
Thanks! 🙏 Updated - [c4ee7f2](https://github.com/commercetools/merchant-center-application-kit/pull/3128/commits/c4ee7f252c01086eced77694f505aeba709d6ca2)